dockerfiles/Dockerfile

```diff
@@ -51,6 +53,17 @@ RUN cd chipyard && \
         export MAKEFLAGS=-"j $(nproc)" && \
         ./scripts/build-toolchains.sh esp-tools 1>/dev/null
 
+
+# Set up FireMarshal
+SHELL ["/bin/bash", "-c"] 
+RUN cd chipyard && \
+        source env.sh && \
+        cd software/firemarshal && \
+        ./init-submodules.sh && \
+        pip3 install -r python-requirements.txt && \
+        marshal build br-base.json 
```

Comment:
To make the Docker image smaller... should we do a `marshal clean br-base.json` to keep the main files but delete the temp files?

Reply:
That would be reasonable, it would save like 500MB. The one that takes a long time is the distro image (br.XXXX.img) which is what we want to avoid.